Disable add button and ripple effect when max number of files added

tehapo opened the pull request:
Fixes #28

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.svg" height="40" alt="Review on Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-upload/32)
<!-- Reviewable:end -->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vaadin/vaadin-upload?pullRequest=32) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

platosha commented:
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[vaadin-upload.html, line 152 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32#-KBm-k7czvpe8kmNcCny:-KBm-k7dQGKyE8wpbGG1:-55787685) ([raw file](https://github.com/vaadin/vaadin-upload/blob/1fbb48042081c0de21b9f63b1e92932bac012a64/vaadin-upload.html#L152)):*
Would make more sense for me to see `files.length` here instead of `.splices`.

---

*[test/adding-files.html, line 116 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32#-KBm1ZMj9DgleX-hkr0s:-KBm1ZMkgyLJAATLjn4c:-451022953) ([raw file](https://github.com/vaadin/vaadin-upload/blob/1fbb48042081c0de21b9f63b1e92932bac012a64/test/adding-files.html#L116)):*
Would make sense to test the `dragover` to not be true when dragging files if max files added.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32)*
<!-- Sent from Reviewable.io -->

tehapo commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[vaadin-upload.html, line 152 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32#-KBm-k7czvpe8kmNcCny:-KBq659uV2JUhE3fgNfI:970109704) ([raw file](https://github.com/vaadin/vaadin-upload/blob/1fbb48042081c0de21b9f63b1e92932bac012a64/vaadin-upload.html#L152)):*
You're right. Fixed.

---

*[test/adding-files.html, line 116 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32#-KBm1ZMj9DgleX-hkr0s:-KBq66vdQ1quvNt-4uwJ:1977649334) ([raw file](https://github.com/vaadin/vaadin-upload/blob/1fbb48042081c0de21b9f63b1e92932bac012a64/test/adding-files.html#L116)):*
True, added a test.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32)*
<!-- Sent from Reviewable.io -->

platosha commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/32#-:-KBr2-4wx_xFogtEZv1L:980077423)*
<!-- Sent from Reviewable.io -->
